packages/expo-yarn-workspaces/index.js

```diff
@@ -25,44 +19,16 @@ exports.createMetroConfiguration = function createMetroConfiguration(projectPath
     ...defaultConfig
   } = getDefaultConfig(projectPath, options);
 
-  let watchFolders;
-  let extraNodeModules;
-
-  const workspaceRootPath = findYarnWorkspaceRoot(projectPath);
-  if (workspaceRootPath) {
-    debug(`Found Yarn workspace root at %s`, workspaceRootPath);
-    watchFolders = [workspaceRootPath];
-    extraNodeModules = {
-      ...getSymlinkedNodeModulesForDirectory(workspaceRootPath),
-      ...getSymlinkedNodeModulesForDirectory(projectPath),
-    };
-  } else {
-    debug(`Could not find Yarn workspace root`);
-    watchFolders = [];
-    extraNodeModules = getSymlinkedNodeModulesForDirectory(projectPath);
-  }
-
   return {
     ...defaultConfig,
     // Search for modules from the project's root directory
     projectRoot: projectPath,
 
-    // Include npm packages from the workspace root, where packages are hoisted
-    watchFolders,
     resolver: {
       ...defaultConfig.resolver,
       // test-suite includes a db asset
       assetExts: [...defaultConfig.resolver.assetExts, 'db'],
 
-      // Include .cjs files
-      sourceExts: [...defaultConfig.resolver.sourceExts, 'cjs'],
-
-      // Make the symlinked packages visible to Metro
-      extraNodeModules,
-
-      // Use Node-style module resolution instead of Haste everywhere
-      providesModuleNodeModules: [],
```

Comment:
Just confirming -- this is no longer needed to opt out of Haste? We don't want `@providesModule` to be picked up at all.

Reply:
This feature is no longer in the Metro repo anywhere https://github.com/search?q=repo%3Afacebook%2Fmetro%20providesModuleNodeModules&type=code